netsuitesdk/errors/errors.py

```diff
@@ -1,18 +1,33 @@
 
 error_reference = {
- "expense_report": 
-    {
+    "expense_report": {
         'category_reference_error': {'regex': r"An error occured in a upsert request: Invalid category reference key \d+ for entity \d+", 'keys': ['expense_category', 'employee']},
```

Comment:
```suggestion
        'category_reference_error': {
            'regex': r"An error occured in a upsert request: Invalid category reference key \d+ for entity \d+",
            'keys': ['expense_category', 'employee']
        },
```
@NileshPant1999 can we format all errors like this, makes them more readable